src/http_server.py

```diff
@@ -0,0 +1,30 @@
+import logging
+from http.server import SimpleHTTPRequestHandler, ThreadingHTTPServer
+from threading import Thread
+
+logger = logging.getLogger(__name__)
+
+PORT = 8080
+HOSTNAME = "localhost"
+
+
+class PythonServer(SimpleHTTPRequestHandler):
+    def do_GET(self):
+        logging.info("request GET / ")
+        self.send_response(200)
+        self.send_header("Content-type", "text/html")
+        self.end_headers()
+        self.wfile.write("GET request for {}".format(self.path).encode("utf-8"))
+
+
+def start():
```

Comment:
```suggestion
def start(logger: Logger):
```
and pass the logger to this function from the `main.py` file (as we have done already)